publiccode.yml

```diff
@@ -0,0 +1,62 @@
+publiccodeYmlVersion: "0"
+name: Integreat CMS
+applicationSuite: Integreat Platform
+url: "https://github.com/digitalfabrik/integreat-cms.git"
+landingURL: "https://integreat-app.de/en/"
+isBasedOn: "https://github.com/django/django.git"
+softwareVersion: "2025.9.0"
+releaseDate: "2025-09-11"
+logo: "https://integreat-app.de/wp-content/uploads/logo-aktuell.svg"
+platforms:
+  - linux
+categories:
+  - content-management
+  - educational-content
+  - translation
+usedBy:
+  - Germany - City of Munich
+  - Germany - City of Dortmund
+developmentStatus: stable
+softwareType: standalone/web
+description:
+  en:
+    shortDescription: The multilingual content management system for the Integreat App
+    longDescription: >
+          The Integreat CMS is a headless content management system for the Integreat App (https://github.com/digitalfabrik/integreat-app).
+          It manages all pages in the same hierarchical structure across all languages. It supports automatic and manual translations.
+    documentation: "https://wiki.integreat-app.com/"
+    apiDocumentation: "https://digitalfabrik.github.io/integreat-cms/api-docs.html"
+    features:
+       - Content Management
+       - Multilingual
+       - Multitenancy
+       - Translation Management
+    screenshots:
+        - "https://integreat-app.de/wp-content/uploads/integreat-cms-screenshot.png"
```

Comment:
AFAICT @JoeyStk added screenshots to the repo at some point. I did not find them immediately. It would probably be better to use screenshots in this repo.

Reply:
#3883

Reply:
Do we need to change the path of the screenshot?